content/maintainers-guide/processes/release/prestashop/final-steps.md

```diff
@@ -7,34 +7,34 @@ aliases:
 
 # Final steps
 
-## 1. Update API stream for 1-click upgrade
+## 1. Update API stream for Update Assistant
 
 {{% notice note %}}
 **This should only be done for stable releases.**
 
 _(i.e. if not for betas and RCs)._
 {{% /notice %}}
 
-{{% notice warning %}}
-**This step requires special rights.**
+Additional changes must be brought to the [Distribution API repository](https://github.com/PrestaShop/distribution-api) 
+to provide the best experience on the Update Assistant module:
 
-Ask a maintainer from the PrestaShop Company with administrative rights on the [PrestaShop API repository](https://stash.prestashop.com/projects/PA/repos/xml) to perform this step.
-{{% /notice %}}
+- Add in the file [releaseNotes.json](https://github.com/PrestaShop/distribution-api/blob/main/resources/json/releaseNotes.json) the new version and the link to its release notes,
+- Update the file [autoupgrade.json](https://github.com/PrestaShop/distribution-api/blob/main/public/json/autoupgrade.json) to make the new release available for update.
+If you needed to release a new version of the module Update Assistant, update the `autoupgrade_recommended` property as well.
+You may have to create a new entry if the new Update Assistant is compliant with a range of PrestaShop version that is unrelated to the existing ones.
 
-Here is an example of a PR that introduces a new version https://stash.prestashop.com/projects/PA/repos/xml/pull-requests/138/diff:
-- you will need to add the new XML checksum file that was generated with the zip archive, this XML file should be named after the version released (ex: for `8.1.2` -> `8.1.2.xml`)
-- the `channel.xml` file needs to be updated, the `branch` node matching the minor version being released must be updated to indicate
-  - the latest patch version
-  - the link to the archive for this patch version
-  - the md5 checksum of this archive file
-  - the link to the build article for this patch version
+Once this step is done, update the Release Tracker GitHub issue by ticking the "Available for update" box.
 
-For new major/minor version you will have to create a new dedicated branch node.
+{{% notice tip %}}
+**Application of changes check.**
 
-Once this step is done, update the Release Tracker GitHub issue by ticking the "Available for upgrade" box.
+_Please verify the contents of https://api.prestashop-project.org/autoupgrade. You might need to wait for the cache to be cleared._
```

Comment:
it's unrelated, but I'm surprised we don't use GH actions to ping CF API to clear the cache :)